platform_versioned_docs/version-23.1.0/enterprise/advanced-topics/use-iam-role.mdx

```diff
@@ -13,22 +13,131 @@ AWS-based customers can configure Tower to interact with other AWS Services like
 
 ## Configure the Tower IAM policy
 
-!!! tip "Assumptions in Provided Policies"
-
-    The provided policies were designed with certain assumptions:
+:::note
+The provided policies were designed with certain assumptions:
 
-    1. **IAM Policy**: Tower and Nextflow should have whole access to identified S3 Buckets.
-    2. **Trust Policy**: The Role should be assumable by EC2, ECS, EKS, and only specifically-named IAM actors.
+1. **IAM Policy**: Tower and Nextflow should have whole access to identified S3 Buckets.
+2. **Trust Policy**: The Role should be assumable by EC2, ECS, EKS, and only specifically-named IAM actors.
 
-    You may wish to limit S3 access to specific prefixes, and/or Role assumption to more specific Platforms.
+You may wish to limit S3 access to specific prefixes, and/or Role assumption to more specific Platforms.
+:::
 
 Create a custom IAM Policy ([Tower-Role-Policy.json](../_templates/Tower-Role-Policy.json)).
```

Comment:
There are AWS IAM policies here: https://github.com/seqeralabs/nf-tower-aws

Unless there is a compelling reason to list another copy of permissions, I suggest purging the code block here and point instead to the `nf-tower-aws` repo.

Reply:
Agreed. Stripped. Also cleaning the AWS repo [here](https://github.com/seqeralabs/nf-tower-aws/pull/12)